test/jdk/javax/swing/JComboBox/6559152/bug6559152.java

```diff
@@ -97,7 +98,7 @@ private static void setupUI() {
     }
 
     private static void test() throws Exception {
-        robot.mouseMove(p.x, p.y);
+        robot.mouseMove(p.x + cb.getWidth() / 2, p.y + cb.getHeight() / 2);
```

Comment:
Since `cb` is swing component, appropriate will be to do 
(Dimension) d = comp.getSize(); inside blockTillDisplayed() which is called in EDT

and use mouseMove(p.x + d.width/2, p.y + d.height/2)